cmd/fyne_demo/screens/advanced.go

```diff
@@ -56,6 +55,11 @@ func AdvancedScreen(win fyne.Window) fyne.CanvasObject {
 		})
 	}
 
+	label := widget.NewLabel("Just type...")
+	head := widget.NewVBox(label,
+		fyne.NewContainerWithLayout(layout.NewGridLayout(2),
+			widget.NewGroup("Generic"),
```

Comment:
This doesn't take advantage of a group, unless group is used for the line, could this be label instead?

Reply:
The group's scrolling is generic unfortunately and it is what was collapsing before.
We could use labels - but I liked the styling :)

Reply:
This is not a problem with my new patch :)